test/end2end/metadata-test-plugin/metadata_test/__init__.py

```diff
@@ -6,5 +8,5 @@ class MetadataPlugin(MetadataTransformer):
     def __init__(self, name="ovos-metadata-test-plugin", priority=15):
         super().__init__(name, priority)
 
-    def transform(self, context: dict | None = None) -> dict:
+    def transform(self, context: Optional[dict] = None) -> dict:
```

Comment:
oh, so it was failing because of python version here?

Reply:
Probably; I forget when that syntax was added, but presumably 3.10 or 3.11 based on the failure